src/system.c

```diff
@@ -123,7 +123,26 @@ void sys_set_seccomp_syscall(bool enable)
  */
 int sys_chk_seccomp_action(uint32_t action)
 {
-	if (action == SCMP_ACT_KILL) {
+	int rc, nr_seccomp;
+	uint32_t kaction = action & SECCOMP_RET_ACTION_FULL;
+
+
+	nr_seccomp = arch_syscall_resolve_name(arch_def_native, "seccomp");
+	if (nr_seccomp != __NR_SCMP_ERROR) {
```

Comment:
Instead of directly querying to see if seccomp(2) is supported, we should use the sys_chk_seccomp_syscall() function so we honor any configured API level settings.

While we are on the topic, I'm wondering if the KILL_PROCESS capability should be optionally gated behind an API level, thoughts?  (Full disclosure: I haven't thought much about it yet, just wanted to mention the idea.)

Reply:
Also ... and I should have mentioned this first, my apologies ... look at what we did to support SCMP_ACT_LOG.

Reply:
Regarding "API levels", if KILL_PROCESS went in at the same time as ACT_LOG, we may be able to just reuse the existing API level ("3", see include/seccomp.h.in).